lib/features/recurring_donations/cancel/cubit/cancel_recurring_donation_cubit.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:givt_app/features/recurring_donations/cancel/repositories/cancel_recurring_donation_repository.dart';
+import 'package:givt_app/features/recurring_donations/overview/models/recurring_donation.dart';
+
+part 'cancel_recurring_donation_state.dart';
+
+class CancelRecurringDonationCubit extends Cubit<CancelRecurringDonationState> {
+  CancelRecurringDonationCubit(
+    this._cancelRecurringDonationRepository,
+  ) : super(CancelRecurringDonationConfirmationState());
+
+  final CancelRecurringDonationRepository _cancelRecurringDonationRepository;
+
+  Future<void> cancelRecurringDonations(
+    RecurringDonation recurringDonation,
+  ) async {
+    emit(CancelRecurringDonationCancellingState());
+    try {
+      await _cancelRecurringDonationRepository.cancelRecurringDonation(
```

Comment:
Same here, probably it's something nice to keep track of, analyse how much is this feature used

Reply:
same here - Logit was added